app/native_crunch_test.go

```diff
@@ -17,10 +17,9 @@ func TestApp_RequestIsCrunchedFile(t *testing.T) {
 		{path: "/foo/bar/myfile.parquet", expected: false},
 		{path: "/foo/bar/myfile.c00.zstd.parquet", expected: false},
 		{path: "/foo/bar/myfile.parquet", expected: false},
-		{path: "/foo/bar/myfile.gr.parquet", expected: true},
-		{path: "/foo/bar/myfile.gr.zstd.parquet", expected: true},
-		{path: "foo/bar/myfile.gr.zstd.parquet", expected: true},
-		{path: "/foo/bar/myfile.c00.gr.zstd.parquet", expected: false},
+		{path: "/foo/bar/myfile.0.gr.parquet", expected: true},
+		{path: "/foo/bar/myfile.zstd.0.gr.parquet", expected: true},
+		{path: "/foo/bar/0.gr.myfile", expected: true},
```

Comment:
Can we also have one without / and without "."
0.gr.myfile
And one without / but just the "."
myfile.0.gr.parquet